tensorflow/lite/kernels/embedding_lookup_test.cc

```diff
@@ -193,6 +268,50 @@ TEST(HybridEmbeddingLookupHybridOpTest, Simple2DTestInt8) {
                   kTestTolerance)));
 }
 
+TEST(HybridEmbeddingLookupHybridOpTest, Simple2DShapeInt8) {
```

Comment:
And same for these three.

Reply:
@alanchiao , thanks for the input, i updated the TC as per your comments.

Regards
Amit